src/main/scala/nl.knaw.dans.easy.multideposit/actions/ValidatePreconditions.scala

```diff
@@ -115,4 +117,13 @@ class ValidatePreconditions(ldap: Ldap) extends DebugEnhancedLogging {
         case false => Failure(InvalidInputException(deposit.row, s"The depositor '$depositorUserId' is not an active user"))
       }
   }
+
+  //TODO This function works as intended when mvnci applied however I did not get any failure message on VM with an invalid UUID
+  def checkBaseRevisionConformsToUUID(deposit:Deposit, base:String): Try[Unit] = {
+    val depositorBaseRevision = base
+    Try(UUID.fromString(depositorBaseRevision)).isFailure match {
```

Comment:
nice check! Didn't think of that when we discussed the issue. Perhaps tidy it up a bit with

```scala
Try(UUID.fromString(...)).recoverWith {
  case e => Failure(InvalidInputException(deposit.row, "base revision is not in UUID format"))
}
```

Besides, please run the formatter on this function.